index.html

```diff
@@ -94,226 +94,24 @@ <h3 class="top-stories">Multi-Post Stories </h3>
                                     <button class="recentBtn" type="button" id="color">Ruby</button>
                                 </div>
                             </div>
-                            <button type="button" class="get-my-resume-btn">See Project </button>
-                        </div>
-                    </div>
```

Comment:
- [ ] When clicked on the `See project` for the recent work (Multi-Post Stories), the popup with details about the project doesn't appears. Kindly check it.

Reply:
resolved and thanks